pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -41,6 +41,28 @@ func checkVersionOfWindowsUpdate() error {
 	return nil
 }
 
+func checkWindowsEdition() error {
+	windowsEditionCmd := `(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID`
+
+	stdOut, _, err := powershell.Execute(windowsEditionCmd)
+	if err != nil {
+		logging.Debug(err.Error())
+		return fmt.Errorf("Failed to get Windows edition")
+	}
+
+	windowsEdition := strings.TrimSpace(stdOut)
+	logging.Debugf("Running on Windows %s edition", windowsEdition)
+
+	switch windowsEdition {
+	case "Professional":
```

Comment:
I spend some time testing this, more than expected ^^ The good news is that on my French "Windows 10 Professionnel", "Professional" was not localized. The relatively bad news is that I tested "ProfessionalN" and "ProfessionalEducationN" Windows releases which both seem to have hyperv support (I managed to get `crc setup` to succeed after disabling the edition check, and running it a second time after reboot).

And I realized that on new systems (without hyperv), we need to mention somewhere that rebooting and re-running `crc setup` is required. After enabling hyperv, a reboot is required, and since hyperv was not enabled, all the subsequent setup steps were not properly done, so crc setup needs to be rerun 
```


        INFO Checking if oc binary is cached
        INFO Checking if podman remote binary is cached
        INFO Checking if goodhosts binary is cached
        INFO Checking if CRC bundle is cached in '$HOME/.crc'
        INFO Checking minimum RAM requirements
        INFO Checking if running as normal user
        INFO Checking Windows 10 release
        INFO Checking Windows edition
        WARN Skipping above check ...
        INFO Checking if Hyper-V is installed and operational
        INFO Installing Hyper-V
        INFO Will run as admin: enable Hyper-V
        ERRO Please reboot your system
        INFO Checking if user is a member of the Hyper-V Administrators group
        INFO Adding user to the Hyper-V Administrators group
        INFO Will run as admin: add user to hyperv admins group
        INFO Checking if Hyper-V service is enabled
        INFO Enabling Hyper-V service
        INFO Will run as admin: enable Hyper-V service
        INFO Checking if the Hyper-V virtual switch exist
        FATA Unable to perform Hyper-V administrative commands. Please reboot your system to complete the setup process
```

Reply:
> And I realized that on new systems (without hyperv), we need to mention somewhere that rebooting and re-running crc setup is required. After enabling hyperv, a reboot is required, and since hyperv was not enabled, all the subsequent setup steps were not properly done, so crc setup needs to be rerun

maybe this was the case i also encountered and needed a reboot, but forgot exactly, good that this is clear now.. will revert the message to `Please reboot your system and run 'crc setup' to complete the setup process`

Reply:
> Interesting, i didn't knew about these editions, so need to look at the whole list of editions that support hyperv and add all of those name to the check..

My feeling is that `strings.HasPrefix(edition, "professional")` would cover these.